src/main/java/stream/flarebot/flarebot/util/RandomUtils.java

```diff
@@ -0,0 +1,13 @@
+package stream.flarebot.flarebot.util;
+
+import java.util.Collection;
+import java.util.Random;
+
+public class RandomUtils {
```

Comment:
Do we really use random enough to constitute having a class for random stuff?

Reply:
Yes to not also deal with random being in the god damn guild JSON, plus it's much quicker